string-manipulation-1/src/get-words.js

```diff
@@ -1,9 +1,10 @@
 /* exported getWords */
 function getWords(string) {
+  var empty = [];
   var newString = string.split(' ');
   for (var i = 0; i < newString.length; i++) {
     if (string.length === 0) {
-      return [];
+      return empty;
     }
   }
   return newString;
```

Comment:
This is not what I meant. You should move the conditional checking if the `string` has no length to the first thing in the function. You don't need a variable holding an empty array, you can just return an empty array like you had before. You also still haven't removed the for loop, please do so.